src/server/game/Handlers/GroupHandler.cpp

```diff
@@ -779,7 +779,7 @@ void WorldSession::BuildPartyMemberStatsChangedPacket(Player* player, WorldPacke
     {
         uint64 auramask = player->GetAuraUpdateMaskForRaid();
         *data << uint64(auramask);
-        for (uint32 i = 0; i < MAX_AURAS; ++i)
+        for (uint32 i = 0; i < MAX_AURAS_GROUP_UPDATE; ++i)
```

Comment:
May I ask why? Not trying to be a smartass, I honestly don't know why it doesn't matter, thought it would save space. Thanks!